### ISSUE TITLE
[question] volunteering to test for group ironman

### ISSUE THREAD

**Issue by creilson** [pwatts6060/slash-swapper#1]
Hey there. Not really an issue, more so a discussion topic - but, it does not look like this repo has that option available.
I was wondering if there's any chance of you looking to re-enable the ability for group ironman accounts to use this plug-in and was curious what you needed tested. I don't mind lending a hand if you are interested, since I was hoping to get my clan to more actively use clan chat (as opposed to friend chat/chat channel), but the obvious reason for your plug-in is old habits die hard. Please let me know if there's anything I can do or if you have an agenda for re-introducing gim support. Thanks in advance.

**Comment by pwatts6060** [comment]
Yeah main issue is I don't have a gim account that is also in a clan. Probably needs an hour or two on one to test and rewrite code to work for them.

**Comment by creilson** [comment]
> Yeah main issue is I don't have a gim account that is also in a clan. Probably needs an hour or two on one to test and rewrite code to work for them.

I have a gim account that is also in a clan, so that works perfectly. I also have programming experience. Let me know if you want to work together to get this tested :) Feel free to contact me on discord "neverlucky#6867"
